CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## v1.8
 
+### Continuous integration
+
+* GitHub Actions CI workflows are now included in the template pipeline
+* Travis CI tests will most likely be deprecated from the next `tools` release
```

Comment:
```suggestion
* Travis CI tests will be deprecated from the next `tools` release
```

Reply:
Didnt want to shoot us in the foot! Ill change that 👍

Reply:
https://github.com/nf-core/tools/pull/483/commits/95f032133fceeb88af2ed73c94dac45bc36a63d7